packages/nx-semantic-release/src/executors/semantic-release/semantic-release.ts

```diff
@@ -47,21 +47,44 @@ export async function semanticRelease(
   };
 }
 
-const applyProjectRoot = (value: string, root: string) =>
-  value.replace('${PROJECT_DIR}', root);
-
-const resolveOptions = (
+const applyTokens = (
```

Comment:
Could you add some units tests for this function? We will probably add more tokens in the future, so would be good to have some basic tests for it at least.

Reply:
I'm admittedly weak in the testing department 😞 But I've added some.  Testing file system paths is challenging between operating systems (specifically unix based and windows), I tried to account for that.  Will probably submit a PR to fix a path issue on windows in project.spec.ts at some point.

Reply:
I've not contributed to a project before where fixed issues in a PR like this.  Am I supposed to "Resolve conversation" after I believe I've completed the work in question?

Reply:
No worries 😁. The tests looks good, lets see if the pipelines will pass now!

> I've not contributed to a project before where fixed issues in a PR like this. Am I supposed to "Resolve conversation" after I believe I've completed the work in question?

Personally I think it's better for the person that created comment to resolve conversation once it's done, but I guess that depends on project.